lisp/mastodon-tl.el

```diff
@@ -408,10 +408,24 @@ Move forward (down) the timeline unless BACKWARD is non-nil."
             "id:%s must be either a string or a number"
             id))))
 
+(defun mastodon-tl--toot-id ()
+  "Find approproiate toot id for toot under point.
+
+If the toot has been boosted use the id found in the
+reblog portion of the toot. Otherwise, use the body of
+the toot. This is the same behaviour as the mastodon.social
+webapp"
+  (let* ((json (mastodon-tl--property 'toot-json))
+         (id (alist-get 'id json ))
+         (reblog (alist-get 'reblog json )))
+    (if reblog
```

Comment:
Should this be wrapped in the new "make it a string" helper function to ensure we return valid ids?